### PR TITLE
Add documents for participant joining related resources

### PR DESCRIPTION
**Pull request by yizhenww**
Add documents for participant joining related resources

### CODE REVIEW

**Comment by yizhenww** [comment]
currently pending sdk verification before proceeding with the doc

File: concepts/changelog.md [hunk 1]
```diff
@@ -13,6 +13,19 @@ For a summary of the value of these API changes, as well as recent tools, compon
 
 ## February 2021
```
Comment (Lauragra) [inline concepts/changelog.md:14]: FYI, this file is now retired in favor of the changelog JSON files. For details about how to author changelog entries, see this topic: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103153/Changelog.

Comment (yizhenww) [reply concepts/changelog.md:14]: seems like the tool can only generate diff between a production schema and a new schema branch, however my schema change is already checked in and has been rolled out to production. Should I remove the change to changelog.md in this PR, and start generating graph doc pr using the tool in future doc update instead?

Comment (yizhenww) [reply concepts/changelog.md:14]: I ended up running the tool against a separate schema version, and manually copying over the changelog json, and the link in changelog.md 

let me know if this looks good

**Review by Lauragra — APPROVED** [review]

**Pull request merged**